manifests/client/host_config_entry.pp

```diff
@@ -377,10 +381,376 @@
     $_gssapiauthentication = false
   }
 
+  $_value = ssh::config_bool_translate($useprivilegedport)
+
   $_name = ssh::format_host_entry_for_sorting($name)
 
-  concat::fragment { "ssh_config_${_name}":
-    target  => '/etc/ssh/ssh_config',
-    content => template("${module_name}/ssh_config.erb")
+  ssh_config{
```

Comment:
This should probably be done as one pure augeas blob instead of a bunch of different calls.

Nothing here is nested or fancy so it should "just work".

That said, that would be a non-breaking change if done later so it can wait.

Reply:
I have always wondered why this module came with a lens, doesn't puppet bundle them themselves, or don't they come in augeasproviders?

Reply:
@jeefberkey  Both lenses were added to work around broken upstream lenses.  The lens in this commit was added because the upstream lenses [didn't manage `HostKeyAlgorithms` ](https://github.com/simp/pupmod-simp-ssh/pull/72/commits/7b49583955e4f0faed2999f9155013e4ef511d83#diff-7cdb01b0b17b7b38afcccd1c16987de1L78).

Fun fact I just noticed: **the sshd.aug lens is no longer used at all**.  It's not in a pluginsync-able module directory and AIO puppet's augeas doesn't read it in the location where it gets established.  It appears that [we can remove it without any consequences](https://gitlab.com/chris.tessmer/pupmod-simp-ssh/pipelines/18921191).

Reply:
@op-ct Neat, thanks! Yay for deleting things